### DISCUSSION TITLE
How can we integrate oxidized with Github for auto backup of devices like cisco switch

### DISCUSSION THREAD

**Discussion by ajkan87** [community/community#76823] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, Good morning,
How can we integrate oxidized with Github for auto backup of devices like cisco switch? I need to integrate the oxidized with Github for backup project, if you can share any document for detailed step that is required to follow at Oxidized, Github and device level, it will great help.

**Comment by ajkan87** [comment]
I have followed the discussion on internet but unable to find a good content with complete details. Looking help from community.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @ajkan87! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

To clarify, are you referring to [the Oxidized project on GitHub](https://github.com/ytti/oxidized)? You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category for you. Good luck!
